### PR TITLE
feat: sl-namespace-init support assigning node

### PR DESCRIPTION
**Pull request by focaaby**
support spec nodeSelector, tolerations, affinity

### CODE REVIEW

**Review by bluesky6529 — DISMISSED** [review]

**Review by bluesky6529 — APPROVED** [review]

**Review by abelite-fang — APPROVED** [review]

File: sl-namespace-init/templates/eventsource.yaml [hunk 1]
```diff
@@ -5,6 +5,18 @@ metadata:
 spec:
   template:
     serviceAccountName: kube-argo
+    {{- with .Values.nodeSelector }}
+    nodeSelector:
+      {{- toYaml . | nindent 6 }}
```
Comment (SL1-AndyChuang) [inline sl-namespace-init/templates/eventsource.yaml:10]: you should use indent not nindent ?
    nodeSelector:
      {{- toYaml . | indent 6 }}
or change to 
    nodeSelector: {{- toYaml . | nindent 6 }}

Comment (focaaby) [reply sl-namespace-init/templates/eventsource.yaml]: Updated

**Review by SL1-AndyChuang — APPROVED** [review]
LGTM

**Review by abelite-fang — APPROVED** [review]
lgtm

**Pull request merged**